images/hubble/Dockerfile

```diff
@@ -0,0 +1,53 @@
+# Copyright Authors of Cilium
```

Comment:
mostly copied and pasted from hubble-relay Dockerfile. please review carefully, i have no idea what's going on here.